Description: Fixes qtloader.js to call the original preRun method with the
argument it's supposed to receive, rather than just with an empty argument list.
This has been fixed after Qt 6.6.1 by using an array for preRun instead, so if
this patch fails to apply in the future, it may have become unnecessary.

--- a/src/plugins/platforms/wasm/qtloader.js
+++ b/src/plugins/platforms/wasm/qtloader.js
@@ -124,7 +124,7 @@
     const originalPreRun = config.preRun;
     config.preRun = instance =>
     {
-        originalPreRun?.();
+        originalPreRun?.(instance);

         throwIfEnvUsedButNotExported(instance, config);
         for (const [name, value] of Object.entries(config.qt.environment ?? {}))
